app/frontend/actions.ts

```diff
@@ -673,53 +674,19 @@ export default (store: Store) => {
 
   const debouncedCalculateDelegationFee = debounceEvent(calculateDelegationFee, 500)
 
-  const validateDelegationAndCalculateFee = () => {
-    const state = getState()
-    const selectedPool = state.shelleyDelegation.selectedPool
-    const delegationValidationError = selectedPool.validationError
-
-    setError(state, {
-      errorName: 'delegationValidationError',
-      error: delegationValidationError,
-    })
-    setState({
-      delegationValidationError,
-    })
-    if (!delegationValidationError) {
-      setState({
-        calculatingDelegationFee: true,
-        gettingPoolInfo: true,
-      })
-      debouncedCalculateDelegationFee(state)
-    } else {
-      setState({
-        shelleyDelegation: {
-          ...state.shelleyDelegation,
-          delegationFee: 0 as Lovelace,
-        },
-      })
-    }
-  }
-
-  const updateStakePoolIdentifier = (
-    state: State,
-    poolHash: string,
-    validationError: any = null
-  ): void => {
+  const updateStakePoolIdentifier = (state: State, poolHash: string): void => {
```

Comment:
I was just totally confused by this function & decided to remove it. @PeterBenc please double-check that it is handled correctly. :pray:

Reply:
I am not suprised, Previously there were multiple stakepool inputs for multiple pools (also reason for `validationError` in being in poolInfo object) and this is just a left over.